composer.json

```diff
@@ -25,15 +25,15 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
Please use `~8.0.0` as mentioned in the issue. That way the package supports PHP up to `8.0.*`.